src/work.tsx

```diff
@@ -10,6 +10,7 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.css';
+import 'styled/theme.css';
```

Comment:
I'm a bit confused. `theme.css` looks like Boostrap; why are we checking it in here instead of reading it from `node_modules`?

Reply:
this is the `front` theme overriding some of the default bootstrap themes

Reply:
if you look at the docs front-v2.2/documentation/index.html#starter-template
they only include the theme and not the bootstrap.css file

Reply:
did a search for styles that bootstrap has that the template doesnt, looks like all the styled from bootstrap are in the template + more! removing the bootstrap.css from the file

Reply:
actually, it looked like the new 4.3 bootstrap came out with 1000 lines of new css. should we still keep the bootstrap.css in case we want to use some of their new class names?

Reply:
i would keep just `theme.css` to avoid any conflicts. the file specifies the bootstrap version at the top

Reply:
👍 we can cherry pick any styles we want from the new version and just add it to the `theme.css` file

Reply:
why not pull in the scss file instead of the css file so we can use set the color/font variables?

Reply:
good call! using scss files